jobs/keepalived/spec

```diff
@@ -0,0 +1,27 @@
+---
+name: keepalived
+
+description: "The keepalived job can be used to add a VRRP IP address to enforce HA on haproxy release"
+
+packages:
+- keepalived
+
+templates:
+  keepalived_ctl: 		bin/keepalived_ctl
+  keepalived.config.erb: 	config/keepalived.config
+
+properties:
+  keepalived.vip:
+    description: Virtual IP V4 address that will be given to master 
+  keepalived.healthcheck_name:
+    description: name of healthcheck that will be performed
```

Comment:
label displayed for the health check. Will appear in keepalive traces e.g. "Keepalived_vrrp[4558]: VRRP_Script(check_haproxy) failed"